### ISSUE TITLE
rails g graphql:mutation does not use base mutation

### ISSUE THREAD

**Issue by lifeiscontent** [rmosolgo/graphql-ruby#2610]
all other generators use a `base` class I'd expect the `rails g graphql:mutation` to do the same.

is there a reason for this? or just an oversight?

**Comment by rmosolgo** [comment]
🤔  Hmm, looking at the template, I'd expect it to use `BaseMutation`: 

https://github.com/rmosolgo/graphql-ruby/blob/9d2bac94cc2c4494b2201d4356306ae26029e8f8/lib/generators/graphql/templates/mutation.erb#L2

What did it generate in your case?

**Comment by newx** [comment]
I tried this on graphql-ruby `1.9.15` and it worked fine.

```
rails g graphql:mutation create_item
       exist  app/graphql/mutations
   identical  app/graphql/mutations/.keep
      create  app/graphql/mutations/base_mutation.rb
        skip  app/graphql/types/mutation_type.rb
add_root_type  mutation
   identical  app/graphql/mutations/create_item.rb
        gsub  app/graphql/types/mutation_type.rb
```

I generated the `app/graphql/mutations/base_mutation.rb` file:

```
module Mutations
  class BaseMutation < GraphQL::Schema::RelayClassicMutation
    argument_class Types::BaseArgument
    field_class Types::BaseField
    input_object_class Types::BaseInputObject
    object_class Types::BaseObject
  end
end
```

And the mutation itself `app/graphql/mutations/create_item.rb`

```
module Mutations
  class CreateItem < BaseMutation
    # TODO: define return fields
    # field :post, Types::PostType, null: false

    # TODO: define arguments
    # argument :name, String, required: true

    # TODO: define resolve method
    # def resolve(name:)
    #   { post: ... }
    # end
  end
end
```

@lifeiscontent are you using version `1.9.15`? 
Can you provide an example output?

**Comment by lifeiscontent** [comment]
@newx yes, in my case my base mutation was not generated during `rails g graphql:install`

and when I run the generator I get something like 

``` ruby
module Mutations
  class CreateItem < GraphQL::Schema::RelayClassicMutation
    # TODO: define return fields
    # field :post, Types::PostType, null: false

    # TODO: define arguments
    # argument :name, String, required: true

    # TODO: define resolve method
    # def resolve(name:)
    #   { post: ... }
    # end
  end
end
```

**Comment by newx** [comment]
@lifeiscontent I was able to reproduce this. I'm going to investigate further and try to fix it.

**Comment by newx** [comment]
@lifeiscontent this is happening because the version was not bumped up after the `install_generator` was updated to generate the `base_mutation.rb` file here c01420f9ccad363f0f1e44bc9b9f0483c9dbc1c3 . Because of it, the code we have on rubygems.org is not updated with graphql-ruby `master` branch.

See here that the version was bumped to `1.9.15` on the commit before it https://github.com/rmosolgo/graphql-ruby/commits/master?after=ac2d2354607d39c2c18711e15489ff9d8314eca3+34

To fix this issue you can point your Gemfile directly to github like this:

```
gem 'graphql', '~> 1.9.15', git: "https://github.com/rmosolgo/graphql-ruby.git"
```

cc @rmosolgo I think this can be closed

**Comment by rmosolgo** [comment]
:+1: thanks for looking into it! I guess it will be fixed in 1.9.16.
